pkg/cloud/devicemanager/allocator.go

```diff
@@ -22,8 +22,7 @@ import (
 
 // ExistingNames is a map of assigned device names. Presence of a key with a device
 // name in the map means that the device is allocated. Value is irrelevant and
-// can be used for anything that NameAllocator user wants.  Only the relevant
-// part of device name should be in the map, e.g. "ba" for "/dev/xvdba".
+// can be used for anything that NameAllocator user wants.
 type ExistingNames map[string]string
```

Comment:
If we don't care about the value, it should be `struct{}`.

Reply:
Although we don't care about the value there _is_ a value: the caller passes in a `map[string]string`, which is not compatible with a `map[string]struct{}`.